tests/unit/synapseclient/unit_test_tables.py

```diff
@@ -1160,13 +1163,19 @@ def test_as_data_frame__list_columns(self):
             'string_list': [['foo', 'bar'], ['wizzle', 'wozzle']],
             'integer_list': [[1, 5], [2, 1]],
             'boolean_list': [[True, False], [False, True]],
+            'date_list': [[1622614955000, 1622614955000], [1622614955000, 1622614955000]],
+            'entity_list': [['syn1235', 'syn1236'], ['syn1235', 'syn1234']],
+            'user_list': [[1111235, 1111233], [1111232, 1111234]],
             'fill': [[], []],
         })
 
         headers = [
             SelectColumn(name='string_list', columnType='STRING_LIST'),
             SelectColumn(name='integer_list', columnType='INTEGER_LIST'),
             SelectColumn(name='boolean_list', columnType='BOOLEAN_LIST'),
+            SelectColumn(name='date_list', columnType='DATE_LIST'),
+            SelectColumn(name='entity_list', columnType='ENTITYID_LIST'),
+            SelectColumn(name='user_list', columnType='USERID_LIST'),
```

Comment:
Will these panda df column names ('string_list', 'integer_list' etc.) be re-used anywhere outside the unit test? I wonder if they should be saved as constants so it's easier to maintain in the future.

Reply:
Can we do something like this for the list of items that need to be cast?
```
elif columnType == 'INTEGER_LIST':
    type = json.loads(field)
    result.append([int(item) for item in type])
```

Reply:
The values returned from synapse are already in integer format:
```
import json
json.loads('[1,2,3]')  
[1, 2, 3]
```

Reply:
If the lists don't need to be cast, I don't see any issue with the function you mentioned. Could you elaborate more about your concern?

Reply:
It was just more a response to your question:

```
result.append([int(item) for item in type])
```

We don't need to do this because the values are cast to integers already.

Reply:
Oh, I am actually curious about why you said previously: 

`this function makes it a bit difficult: https://github.com/thomasyu888/synapsePythonClient/blob/add-entity-user-list/synapseclient/table.py#L499-L536.`

Reply:
Ahh, I was hoping that by maintaining a global variable of all the "synapse list columns" I wouldn't need to update it manually in other areas, but due to this function, it seems like I still have to.

Reply:
I see. We only need to update 2 places rather than many other places. So I think this is still an improvement.